docs/config-file.md

```diff
@@ -129,13 +145,25 @@ These configurations apply to Glee itself, rather than any specific protocol.
 
 |Field|Description|
 |---|---|
-|kafka.auth| Kafka authentication configuration|
-|kafka.auth.key | Kafka Broker Key
-|kafka.auth.cert| Client certificate
-|kafka.auth.clientId| Kafka client Id for authentication
-|kafka.auth.rejectUnauthorized | Boolean flag for accepting the valid SSL certificates
-|kafka.auth.username| The username to use during authentication.
-|kafka.auth.password| The password to use during authentication.
+|kafka.authentication| Kafka authentication configuration|
+|kafka.authentication.key | Kafka Broker Key
+|kafka.authentication.cert| Client certificate
+|kafka.authentication.clientId| Kafka client Id for authentication
+|kafka.authentication.rejectUnauthorized | Boolean flag for accepting the valid SSL certificates
+|kafka.authentication.username| The username to use during authentication.
+|kafka.authentication.password| The password to use during authentication.
+
+### Http Server
+
+|Field|Description|
+|--|--|
+|http.server|Http server-specific configurations|
+|http.client|Http client-specific configurations|
+|http.server.adapter| The Glee adapter to use for the Http server. Defaults to a "native" Http implementation.|
+|websocket.server.port| The port to use when binding the Http server. This is useful when your server is behind a proxy and the port exposed for consumption is not the same as the port your application should be bound to. Defaults to the port specified in the selected AsyncAPI server.|
+|http.client.authetication| Authentication variables for client|
```

Comment:
```suggestion
|http.client.auth| Authentication variables for client|
```

it would be best if you defined the auth object for HTTP, like token-based or basic authentication